documentation/src/docs/pravega-concepts.md

```diff
@@ -146,7 +146,7 @@ space is then divided into a number of partitions, corresponding to the number
 of Stream Segments. Consistent hashing determines which Segment an Event is
 assigned to.
 
-### AutoScaling: The number of Stream Segments can vary over time
+### AutoScaling: Number of Stream Segments can vary over time
```

Comment:
Revert this change, please.

Reply:
`### AutoScaling: The number of Stream Segments can vary over time `
is referred with the following link

* .md 
`pravega-concepts.md#autoscalingthenumber-of-stream-segments-can-vary-over-time`

* .html
 `pravega-concepts#autoscaling-the-number-of-stream-segments-can-vary-over-time`

In key-features.md, we look for `pravega-concepts.md#autoscalingthenumber-of-stream-segments-can-vary-over-time`,  which is resolved , when we are in  markdown format,

But when we look for the same in `key-features/index.html` , we will unable to resolve
`pravega-concepts#autoscalingthenumber-of-stream-segments-can-vary-over-time`
as that doesn't exist.

To solve that problem i had to drop `The`.

Reply:
Change the subtitle then: `Auto-scaling: Varying the number of stream segments over time`.

Reply:
done.